ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py

```diff
@@ -1272,6 +1272,43 @@ def test_numpy_instance_truediv__(
         frontend=frontend,
     )
 
```

Comment:
ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py:1275:1: E302 expected 2 blank lines, found 1

Reply:
Made the change!